src/sandbox/messenger/api.ts

```diff
@@ -34,6 +36,11 @@ const loadSandbox = pMemoize(async () => {
 
 export default loadSandbox;
 
+async function isSandboxed(): Promise<boolean> {
```

Comment:
If you feel that this is called too often, it can be `once`'d 
```suggestion
const isSandbox = once(async (): Promise<boolean> => {
```

Reply:
@fregante yes, I think we'd want to wrap in once. There's no use case for dynamically togging on and off on the same page session